libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaDownloader.kt

```diff
@@ -199,7 +199,7 @@ object SchemaDownloader {
         isDeprecated
         deprecationReason
       }
-      inputFields {
+      inputFields(includeDeprecated: true {
         ...InputValue
```

Comment:
You'll also want to update the `InputValue` fragment to include `isDeprecated` and `deprecationReason`

Reply:
Done.